ablytest/pagination.go

```diff
@@ -73,14 +73,32 @@ func testPagination(request reflect.Value, expectedItems []interface{}, perPage
 		var gotItems []interface{}
 		pageNum := 1
 		expectedFullPages := len(expectedItems) / perPage
+		pageMap := make(map[int]map[string]bool)
```

Comment:
```suggestion
		pageMap := make(map[int]map[string]bool, expectedFullPages)
```
As we know how many pages we are expecting, we should probably provide a size when making the page map. This will let the compiler know to create a map which is allocated at least enough memory to hold the number of expected pages before memory is allocated again.